templates/base/head.tmpl

```diff
@@ -130,9 +130,13 @@
 {{template "custom/header" .}}
 </head>
 <body>
+	{{template "custom/body_outer_pre" .}}
```

Comment:
`custom/templates/custom/` ? :trollface:

Reply:
@bkcsoft just following the already existing templates `custom/header.tmpl` and `custom/footer.tmpl`.

Reply:
```
root@dchi: ~
<o> # tree /var/lib/docker/volumes/demo_gitea/_data/templates
/var/lib/docker/volumes/demo_gitea/_data/templates
└── base
    ├── head.tmpl
    ├── head.tmpl.master
    ├── head.tmpl.new
    └── head.tmpl.patch
```

:trollface:

Reply:
:trollface: